redirect.json

```diff
@@ -296,6 +296,12 @@
     "description": "show your people",
     "introduced": "2021.3"
   },
+  {
+    "redirect": "repairs",
+    "name": "Repairs",
+    "description": "show your Home Assistant Repairs",
```

Comment:
Shouldn't repairs be lower case, and should Home Assistant be there?
Now it's not consistent with the others

Reply:
Not sure tbh